### DISCUSSION TITLE
Can i change device "ID" ?

### DISCUSSION THREAD

**Discussion by lurendrejer** [Koenkk/zigbee2mqtt#13244] [General]
Hi.

Is it possible to change the #/ID -number of a device?

This is what it looks like in the database
![image](https://user-images.githubusercontent.com/30922656/180368727-1fffb89a-7f36-4a01-bbb5-45ff519ffaa7.png)

**Comment by Koenkk** [comment]
yes you can change it (just change the number in the database.db file) but I don't see any reason why you want to do this.

**Reply by lurendrejer** [reply]
Thank you.
While I was moving devices from deconz I tried the Touchlink include and it picked up a bulb.... It's a simple case of my OCD not liking that three bulbs from a single fixture doesn't have adjescent ID's :)

In the default view on the frontend - the bulbs is shown by their ID.

I was worried that changing the ID would somehow mess with relations to data elsewhere.
